deploy/teardown_resources.sh

```diff
@@ -7,31 +7,59 @@ set -euxo pipefail
 set_namespace $CONJUR_NAMESPACE_NAME
 
 configure_cli_pod
+
+# Helm Chart clean-up
+rm -f conjur.pem
+
 if [ "${DEV}" = "false" ]; then
-  $cli_with_timeout "exec $(get_conjur_cli_pod_name) -- conjur variable values add secrets/test_secret \"supersecret\""
+  pushd ../../../secrets-provider/ci
+else
+  pushd ../secrets-provider/ci
+fi
+    find . -type f ! -name '*template.yaml' -delete
+popd
+
+# Delete Helm Chart if already exists
+set_namespace $APP_NAMESPACE_NAME
+if [ "$(helm ls -aq | wc -l | tr -d ' ')" != 0 ]; then
+  helm delete $(helm ls -aq)
 fi
 
+set_namespace $CONJUR_NAMESPACE_NAME
+
+$cli_with_timeout "exec $(get_conjur_cli_pod_name) -- conjur variable values add secrets/test_secret \"supersecret\""
+
 set_namespace $APP_NAMESPACE_NAME
 
 $cli_with_timeout "delete secret dockerpullsecret --ignore-not-found=true"
 
 $cli_with_timeout "delete clusterrole secrets-access-${UNIQUE_TEST_ID} --ignore-not-found=true"
```

Comment:
didn't we change this to role? if not - why?

Reply:
this is part of our previous tests (TEST_ID 1 - 16). 
As we spoke about earlier, it is loaded in a different `.sh.yml` format and so any sort of refactoring will be a whole separate task

Reply:
do we have such a task?

Reply:
Those 'sh.yml' are for tests primarily so I think it is worth a discussion with the team. Since we are moving to BDD anyways it might be worth it to weigh this and see if we still need it